src/OpenTelemetry.Exporter.OneCollector/CommonSchema/CommonSchemaMetadataFieldPrivacyClassificationType.cs

```diff
@@ -0,0 +1,93 @@
+// <copyright file="CommonSchemaMetadataFieldPrivacyClassificationType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.OneCollector;
+
+/// <summary>
+/// Describes the Common Schema field privacy classification type.
+/// </summary>
+public enum CommonSchemaMetadataFieldPrivacyClassificationType
```

Comment:
Why do we want these in the exporter?

Reply:
They are necessary if we want to make a really friendly API. I just pushed a change which drops them and instead exposes the underlying computed value. That will push the heavy lifting for sorting out how to compute the value onto consumers. Works for me if you would prefer it that way.

Reply:
I think privacy classification is independent of exporter and should be handled elsewhere. More importantly, OpenTelemetry's philosophy is that instrumentation should be agnostic to backend.